dot/parachain/availability-distribution/availability_distribution.go

```diff
@@ -163,10 +165,37 @@ func (ad *AvailabilityDistribution) handleChunkFetchingRequest(
 	return response, nil
 }
 
-//nolint:unused
 func (ad *AvailabilityDistribution) handlePoVFetchingRequest(
-	who peer.ID,
+	_ peer.ID,
```

Comment:
why keep the param then?

Reply:
Because the function signature is defined in [`network.RequestHandler`](https://github.com/ChainSafe/gossamer/blob/b19ac8da6790c1994c17486923e6ce52c8f3a78e/dot/network/request_response.go#L22) which is passed to [the network service](https://github.com/ChainSafe/gossamer/blob/b19ac8da6790c1994c17486923e6ce52c8f3a78e/dot/network/service.go#L595).